testsuite/MDAnalysisTests/data/Amber/ache.prmtop

```diff
@@ -1439,3 +1439,42 @@ modified Bondi radii (mbondi)
   8.50000000E-01  8.50000000E-01  7.20000000E-01  8.50000000E-01  8.50000000E-01
   7.90000000E-01  8.50000000E-01  8.50000000E-01  8.50000000E-01  7.20000000E-01
   8.50000000E-01  8.50000000E-01
+%FLAG RESIDUE_NUMBER
+%COMMENT Residue number (resSeq) read from PDB file; DIMENSION(NRES)
+%COMMENT Residue insertion code (iCode) not present in PDB file
+%COMMENT If present: %FLAG RESIDUE_ICODE, %FORMAT(20a4)
+%FORMAT(20I4)
+   1   2   3   4   5   6   7   8   9  10  11  12  13  14
+%FLAG RESIDUE_CHAINID
```

Comment:
I'd rather keep the old test file and make a new one _with_ the additions. Then test both.